beacon-chain/blockchain/process_block.go

```diff
@@ -558,6 +559,17 @@ func (s *Service) isDataAvailable(ctx context.Context, root [32]byte, signed int
 	// The gossip handler for blobs writes the index of each verified blob referencing the given
 	// root to the channel returned by blobNotifiers.forRoot.
 	nc := s.blobNotifiers.forRoot(root)
+	nextSlot := slots.BeginsAt(signed.Block().Slot()+1, s.genesisTime)
+	// Avoid underflow if Now is somehow after the next slot start.
```

Comment:
I think instead of an error it's fine to simply not log these cases:
- On init sync it won't be noisy. 
- For very late blocks anyway the logs already show all logs out of order, so the context deadline error we already have is good enough.

Reply:
init-sync won't go into this path so no issue there, but I can remove the else error log you mention since we'll see the deadline anyway.